Add apt repo urls of tur repository to download dependencies.
--- a/scripts/properties.sh
+++ b/scripts/properties.sh
@@ -47,18 +47,30 @@
 	https://packages-cf.termux.dev/apt/termux-main
 	https://packages-cf.termux.dev/apt/termux-root
 	https://packages-cf.termux.dev/apt/termux-x11
+	https://tur.kcubeterm.com
+	https://tur.kcubeterm.com
+	https://tur.kcubeterm.com
+	https://tur.kcubeterm.com
 )
 
 TERMUX_REPO_DISTRIBUTION=(
 	stable
 	root
 	x11
+	tur-packages
+	tur-packages
+	tur-packages
+	tur-packages
 )
 
 TERMUX_REPO_COMPONENT=(
 	main
 	stable
 	main
+	tur
+	tur-on-device
+	tur-continuous
+	tur-multilib
 )
 
 # Allow to override setup.
